test-result-summary-client/src/Settings/Settings.jsx

```diff
@@ -250,6 +290,17 @@ export default class Settings extends Component {
                         <div className="divider" />
                         <Button type="primary" onClick={this.handleSubmit} >Submit</Button>
                     </div>
+                    <Modal
+                        title="Select how to delete"
+                        onCancel={this.modalCancel}
+                        style={{ top: '20%' }}
+                        visible={this.state.showModal}
+                        footer={null}
+                        >
+                        <Button type="primary" size="large" onClick={this.deleteBuildsKeepData} >Keep builds in database</Button>
+                        <div className="divider" />
+                        <Button type="primary" size="large" onClick={this.deleteBuildsData} >Delete builds in database</Button>
```

Comment:
- Instead of having two functions `deleteBuildsKeepData ` and `deleteBuildsData `, can we have one function `deleteBuilds` and pass in a flag to indicate delete data or not?
- It is better to keep the buttons simple - Yes/No. We can add text:
```
In addition to deleting the build URL from the Build Monitoring List, do you also want to delete the actual build data from the database? 
```
- `title="Advanced Option"`